CMake/Dependencies/libkvspic-CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ include(ExternalProject)
 # clone repo only
 ExternalProject_Add(libkvspic-download
 	GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-pic.git
-    	GIT_TAG           76223690da1329d733309e2a52575d6a98a68981
```

Comment:
Unintended change since there is a PR for it already?

Reply:
Yeah at the time I put out this PR, the other one hadn't been merged -- I'll update branch before merging.